### ISSUE TITLE
How can I use nnUnet for motion artifacts removal task of brain mri

### ISSUE THREAD

**Issue by ramimhasan02** [MIC-DKFZ/nnUNet#2496]
I am working with brain mri images. I want to use nnunet to build a model to remove motion artifacts. so i have uploaded the ground truth files which are whole 3d brain images in the label dataset and the motion simulated images in the training dataset. now in the json file i have described two labels . but when i ran the plan_and_preprocess command i got error saying that i have 520 labels in my files. what should I do now?

**Comment by seziegler** [comment]
Hi, is the issue still relevant?
